--- origsrc/zziplib-0.13.60/configure.ac	2010-05-08 08:36:29.000000000 -0500
+++ src/zziplib-0.13.60/configure.ac	2010-12-31 01:24:59.655750700 -0600
@@ -294,7 +294,7 @@ AC_MSG_CHECKING([link options])
 ZZIPLIB_LDFLAGS=""
 test ".$can_build_shared" != ".no" && ZZIPLIB_LDFLAGS="--export-dynamic"
 RESOLVES=" # "
-case "$host_os" in mingw*)
+case "$host_os" in cygwin*|mingw*)
     ZZIPLIB_LDFLAGS="-no-undefined -export-all-symbols -mconsole"
     RESOLVES=' '
 ;; esac
