--- origsrc/zziplib-0.13.62/configure.ac	2012-03-11 15:28:35.000000000 -0500
+++ src/zziplib-0.13.62/configure.ac	2013-05-29 21:32:00.820207500 -0500
@@ -296,6 +296,9 @@ case "$host_os" in mingw*)
     # we may assume that the backend is gnu-ld but DLLs need full resolution
     ZZIPLIB_LDFLAGS="${wl}-no-undefined ${wl}-export-all-symbols -mconsole"
     RESOLVES=' '
+;; cygwin*)
+    ZZIPLIB_LDFLAGS="-no-undefined"
+    RESOLVES=' '
 ;; darwin*)
     # darwin never uses gnu-ld but the linker can do it anyway
     ZZIPLIB_LDFLAGS="-export-dynamic"
